arviz/plots/ppcplot.py

```diff
@@ -68,38 +103,92 @@ def plot_ppc(
                 '`data` argument must have the group "{group}" for ppcplot'.format(group=group)
             )
 
-    if kind.lower() not in ("density", "cumulative"):
-        raise TypeError("`kind` argument must be either `density` or `cumulative`")
+    observed = data.observed_data
+    posterior_predictive = data.posterior_predictive
+
+    #if density_type.lower() not in ("pdf", "cdf"):
+    #    raise TypeError("`kind` argument must be either `pdf` or `cdf`")
 
     if data_pairs is None:
         data_pairs = {}
 
-    observed = data.observed_data
-    posterior_predictive = data.posterior_predictive
+    data_threshold = 30
```

Comment:
probably this should be an argument that defaults to 30

Reply:
I agree. See my comment below. This is now num_pp_samples.